course_info/static/course_info/js/controllers/PeopleController.js

```diff
@@ -105,15 +107,22 @@
                 : b.active > a.active;
         };
         $scope.disableAddUserButton = function(){
-            if ($scope.searchResults.length > 0 ){
-                return (!$scope.selectedResult.id);
-            }
-            else if ($scope.searchTerm.length > 0){
+
+            /*
+            * this method will disable the button when a search is in progress
+            * it will enable the button if multiple results are found and one
+            * is selected.
+            * */
+            if ($scope.searchTerm.length > 0 && ! $scope.searchInProgress){
```

Comment:
I think you can just add a check for search in progress at the beginning to the previous logic.
```
if ($scope.searchInProgress) {
    return true;
}
else if ($scope.searchResults.length > 0) {
    return (!$scope.selectedResult.id);
}
else if ($scope.searchTerm.length > 0) {
    return false;
}
else {
    return true;
}
```